### ISSUE TITLE
[Feature]: At present, the document expr string matches only the prefix match ‘like ‘prefix%’’, how to check the full match? 

### ISSUE THREAD

**Issue by eechenyi** [milvus-io/milvus#25252]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

At present, the document expr string matches only the prefix match, how to check the full match? For example, the value of VCHAR is ccc, aaa, bbb, so it is found that the value contains the data of aaa

**Comment by xiaofan-luan** [comment]
currently we don't support full match.

is there a specific case you want to do like that.

/assign @zhagnlu

**Comment by eechenyi** [comment]
Yes, I need to query whether a scalar field string contains a certain substring

**Comment by xiaofan-luan** [comment]
> Yes, I need to query whether a scalar field string contains a certain substring

could it be using list<string> contains? wild card matching would be doable but it is very slow

**Comment by xiaofan-luan** [comment]
don't really recommend to do so.
